src/test/java/com/google/api/codegen/testdata/python_no_path_templates.baseline

```diff
@@ -132,6 +132,11 @@ class NoTemplatesServiceApi(object):
         and trailing
         whitespace.
 
+        Example:
+          >>> from google.example.v1 import no_path_templates_pb2
+          >>> api = NoTemplatesServiceApi()
```

Comment:
I added a FIXME comment to revisit this and Jacob's comment (https://goto.google.com/huezs) before finalizing this PR.